src/lexer/token_mapping.cpp

```diff
@@ -33,97 +34,98 @@ namespace details {
  * in multiple context and hence we are keeping original names.
  * Once we finish code generation part then we change this.
  */
-static std::map<std::string, TokenType> keywords = {{"VERBATIM", Token::VERBATIM},
```

Comment:
I've used `clang-format-9` for the formatting, however it might not be the version that changed the formatting but the addition of `const` in the beginning. Then `{{"VERBATIM", Token::VERBATIM},` doesn't fit in the end of the line and the other entries cannot be placed underneath it